baggers.js

```diff
@@ -13992,6 +13992,39 @@ var data = {
         "cities": [
             "Toulouse"
         ]
+    },
+    {
+      "since": "2017-01-01",
+      "name": "Harrich S",
```

Comment:
Well. It goes against what we wrote in our docs: https://github.com/brownbaglunch/bblfr_data#speakers-dataset

> `name`: your full name "First Last"

Is Harrich your first name or last name?